tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/types/FiniteTypeTest.java

```diff
@@ -40,11 +41,13 @@
 public class FiniteTypeTest {
 
     private CompileResult result;
+    private CompileResult negativeResult;
```

Comment:
This can be a local variable in `testTypeDefinitionsWithNullNegative`.

Reply:
Changed